pkg/chains/formats/slsa/attest/attest.go

```diff
@@ -77,10 +79,18 @@ func Invocation(source *v1beta1.RefSource, params []v1beta1.Param, paramSpecs []
 	}
 
 	i.Parameters = iParams
-
 	environment := map[string]map[string]string{}
 
 	annotations := map[string]string{}
+	var meta metav1.Object
+	switch o := obj.GetObject().(type) {
+	// nolint:staticcheck
+	case *v1beta1.PipelineRun:
+		meta = o.GetObjectMeta()
+	// nolint:staticcheck
+	case *v1beta1.TaskRun:
+		meta = o.GetObjectMeta()
+	}
```

Comment:
I think this is not needed. As it's interface, we can just `obj.GetAnnotations()` and `obj.GetLabels()`.

Reply:
Good catch! I fixed it. Thanks